modules/govuk/manifests/apps/static.pp

```diff
@@ -51,20 +51,22 @@
   $ga_universal_id = undef,
   $redis_host = undef,
   $redis_port = undef,
+  $unicorn_worker_processes = undef
```

Comment:
It would be good to add some documentation for this field similar to the others above.

Reply:
Thanks @thomasleese! Have amended.